.github/workflows/rust.yml

```diff
@@ -22,8 +25,8 @@ jobs:
         toolchain: stable
     - uses: actions/checkout@v3
     - uses: Swatinem/rust-cache@v2
-    - name: Install dependencies
-      run: sudo apt-get install clang protobuf-compiler
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v1
```

Comment:
There are explicit dependencies. `Clang` isn't mentioned but it's required for C++ dependencies like rocksdb

Reply:
Somehow it works :confused: I think clang is installed somewhere in rust installation step

Reply:
Let's keep it as is and fix it when something will go wrong